src/ext/java/org/opentripplanner/ext/fares/impl/OrcaFaresData.java

```diff
@@ -249,22 +252,35 @@ class OrcaFaresData {
     sEntry("Southworth-VashonIsland", 6.10f, 3.05f)
   );
 
-  private static Map.Entry<String, Map<FareType, Float>> entry(
+  private static Map.Entry<String, Map<FareType, Money>> entry(
     String name,
     float regularFare,
     float electronicRegularFare
   ) {
     return Map.entry(
       name,
-      Map.of(FareType.regular, regularFare, FareType.electronicRegular, electronicRegularFare)
+      Map.of(
+        FareType.regular,
+        Money.ofFractionalAmount(USD, regularFare),
+        FareType.electronicRegular,
+        Money.ofFractionalAmount(USD, electronicRegularFare)
```

Comment:
These should be `usDollars(<fare>)`.

Reply:
d5c860ca5c1330d9ee43a32d18b659580a5f5f35